diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/constants/Warnings.kt

```diff
@@ -264,7 +299,7 @@ enum class Warnings(
 
     companion object {
         val names by lazy {
-            values().map { it.name }
+            entries.map { it.name }
```

Comment:
is not the entries the `key` `value` pairs?

Reply:
kotlin 1.9 suggests to migrate to `entries` since `values()` will be removed

Reply:
and `entries` is full replacement for values() -- same semantic